### DISCUSSION TITLE
What can be done to prevent the airplane from turning into the wind?

### DISCUSSION THREAD

**Discussion by d-kaliuta** [JSBSim-Team/jsbsim#1144] [Q&A]
Hello!, I am trying to simulate the movement of a c172p airplane with a side wind of 16 fps. And I get an unexpected result at the beginning of the simulation, when the airplane is still on the ground, moving at a speed of 10-20 kilometers per hour. 

The airplane is turning into the wind. 

I read the Issues and noticed that a similar problem is described here https://github.com/JSBSim-Team/jsbsim/issues/89, but the solution described there did not help.

The wind parameters I am using are:
<property value="16"> atmosphere/cosine-gust/magnitude-ft_sec </property>
<property value="2"> atmosphere/cosine-gust/startup-duration-sec </property>
<property value="10000000"> atmosphere/cosine-gust/steady-duration-sec </property>
<property value="0.1"> atmosphere/cosine-gust/end-duration-sec </property>
<property value="3"> atmosphere/cosine-gust/frame </property>
<property value="1.0"> atmosphere/cosine-gust/X-velocity-ft_sec </property>
<property value="1.0"> atmosphere/cosine-gust/Y-velocity-ft_sec </property>
<property value="0.0"> atmosphere/cosine-gust/Z-velocity-ft_sec </property>

Runway heading 270 degrees

configuration file 
[c172p.xml.zip](https://github.com/user-attachments/files/16689426/c172p.xml.zip)

jsbsim output
[cessna.csv](https://github.com/user-attachments/files/16689405/cessna.csv)

**Comment by seanmcleod** [comment]
Your csv file has something like 36 columns, but there are a number of useful states missing, in particular I didn't see heading included. So difficult to see how much weather cocking there is.

Given the velocities you mention for the wind and ground speed, you're talking about a beta angle on the order of 45 deg.

Given that the aircraft is directionally stable there will be a weather cock effect. Check out the forces and moments that depend on beta. Versus the moment of inertia and the friction forces from the gear.

**Reply by d-kaliuta** [reply]
We send you a file with all the parameters. The airplane is on the runway, with low thrust, moving at a speed of up to 20 kilometers per hour. The airplane is slowly drifting into the wind until the airplane's heading is aligned with the wind.

[cessna_full.csv](https://github.com/user-attachments/files/16690932/cessna_full.csv)

**Comment by seanmcleod** [comment]
So what do you see if you plot the aerodynamic forces and moments that use beta and plot the gear forces?

**Reply by d-kaliuta** [reply]
In the air, the airplane behaves adequately.   Is there anything we can do to make sure that on the ground the friction forces (or other forces) are able to counteract the YAW turning moment?  We do not specify friction forces in the xml configuration file. How can we make the model adequate on the ground with wind?

**Comment by seanmcleod** [comment]
> We do not specify friction forces in the xml configuration file....

Are you sure?

https://github.com/JSBSim-Team/jsbsim/blob/4d93022dcc1f44cbe3447ce90044a44dffac447d/aircraft/c172p/c172p.xml#L113-L115

**Reply by d-kaliuta** [reply]
In order for the friction force to provide more resistance than the aerodynamic Yaw, do we need to increase all three friction indicators, or just one of them?
